src/components/ImportRecordingFromSpikeForest.tsx

```diff
@@ -1,375 +0,0 @@
-import { Button, CircularProgress, FormControl, FormGroup, Input, InputLabel, makeStyles, MenuItem, Select } from '@material-ui/core';
```

Comment:
Removing all code for importing within the GUI. Imports will take place in Python scripts for now.